extras/ribbon.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2015 Petr Shevtsov (petr.shevtsov@gmail.com)
+ * MIT-licensed (http://opensource.org/licenses/MIT)
+ *
+ * Plugin options:
```

Comment:
It would be helpful to have a high-level description of this plugin and what it does, e.g. what is a "palette interval"?